docs/source/installation.rst

```diff
@@ -50,7 +50,8 @@ To install a recent version of Python, we recommend to install Anaconda.
 Anaconda offers a user-friendly Python environment. Install Anaconda from 
 the website `Anaconda for MacOS <https://docs.anaconda.com/anaconda/install/mac-os/>`__.
 Make sure to add Anaconda to your PATH environment variable when asked.
-Afterwards, the step above to check if Python is installed.
+Afterwards, the step above to check if Python is installed. If Python is installed properly you can open Python by typing 'python' in 'terminal'.
```

Comment:
'python' opens the interactive console. 'python -V' is a good check and shows the version